### ISSUE TITLE
Alarm triggered does not show as state in Home Assistant 

### ISSUE THREAD

**Issue by superpower10** [tschamm/boschshc-hass#77]
1. Enabled the Bosch Alarm System
2. Wait set time
3. Then I opened the door with Bosch sensor
4. Bosch Smart Home App sends notifications that an Alarm has been triggered

This Trigger does not appear / arrive in the State of Entity „alarm_control_panel.bosch_alarm_system“ in Home Assistant.

According to Home Assistant Atomations it should be possible:
platform: device
device_id: xxxxx
domain: alarm_control_panel
entity_id: alarm_control_panel.bosch_alarm_system
type: triggered

Reason I need this:
Automation that send notification with URL to Camera App on iOS so I can look what’s going on. 

I am not using Bosch cameras. They do not meet my expectations.

**Comment by saxn-paule** [comment]
I can confirm this.
![image](https://user-images.githubusercontent.com/8352690/216818636-d05d2462-82e3-4e74-879b-7ed3b59ffe62.png)

Alarm is triggered an siren makes noise:
![image](https://user-images.githubusercontent.com/8352690/216818774-5044d04f-01b4-49f9-9914-de9c5e1f4c84.png)

**Comment by tschamm** [comment]
Fixed with the latest release 0.4.49

**Comment by superpower10** [comment]
@tschamm 
I hoped so but today the window sensors triggered an alarm because the dogs where barking.

But sadly the Alarm trigger didn‘t show up in Home Assistant.

That’s how I check it:

```
platform: device
domain: alarm_control_panel
entity_id: alarm_control_panel.bosch_alarm_system
type: triggered

```

**Comment by lu-mueller** [comment]
@tschamm  I still have the problem with version 0.4.50

**Comment by lu-mueller** [comment]
Fixed with the release 0.4.58
